### PR TITLE
Updated Using on Windows topic

### PR DESCRIPTION
**Pull request by stevehoag**
linked to RTM resources

### CODE REVIEW

**Comment by richlander** [comment]
I don't think that OPS likes reverts. Can you rebase the change so that it doesn't have any reverts anymore?

**Comment by mairaw** [comment]
The failure was a space disk issue earlier. I'm trying to figure out how to kick off the builds again for each PR

**Comment by richlander** [comment]
Does this PR overlap with #673?

**Comment by stevehoag** [comment]
No, it doesn't overlap with #673. I did rebase, but for some reason it was still picking up files from the earlier commit - reverting seemed to fix that problem.

File: docs/core/tutorials/using-on-windows.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ A solution using only .NET Core projects
         Assert.Equal(42, new Thing().Get(42));
     }
     ```
-    * Optionally, delete the `Program.cs` file from the **TestLibrary** project, and remove `"buildOptions": {"emitEntryPoint": true}` from `project.json`.
+    * Optionally, delete the `Program.cs` file from the **TestLibrary** project, and remove `"buildOptions": {"emitEntryPoint": true,` from `project.json`.
```
Comment (svick) [inline docs/core/tutorials/using-on-windows.md]: Was this supposed to be `"buildOptions": {"emitEntryPoint": true},`?

Comment (stevehoag) [inline docs/core/tutorials/using-on-windows.md]: @svick  Thanks for catching this. A byproduct of cut and paste not working quite right on the new site.

**Comment by stevehoag** [comment]
Closing and reopening to kick off build

**Comment by stevehoag** [comment]
Closing; will try with a new branch

**Pull request closed without merging**